packages/core/src/plugin/default-search-plugin/types.ts

```diff
@@ -35,6 +37,8 @@ export interface DefaultSearchPluginInitOptions {
      * @default false
      */
     bufferUpdates?: boolean;
+
```

Comment:
Can you add a documentation block here? It should have `@description`, `@since 1.6.0` and `@default undefined` tags. If you are not sure what to write for the description let me know and I can assist.